servers/curated-corpus-api/schema-shared.graphql

```diff
@@ -93,6 +93,11 @@ type Section {
   """
   active: Boolean!
   """
+  Indicates whether or not a Section is fully disabled from display on NewTab. Can only  be controlled
+  in the admin tool.
+  """
+  disabled: Boolean!
```

Comment:
i think we could keep this in the admin schema only. the purpose of the field is to prevent any `disabled` sections from showing up in the public query, so this field would never be anything but `false` publicly.

also - best to keep the public schema as light as possible. public clients won't have any use for this field.